File: Practice/AllDomains/Tutorials/30DaysOfCode/Day3IntroToConditionalStatements.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
class Solution 
{
	/*	author:@shivkrthakur  */
	static void Main(String[] args) 
	{
        int n = Convert.ToInt32(Console.ReadLine());
        if(n % 2 != 0)
        {
            Console.WriteLine("Weird");
        }
        else if(n >= 2 && n < 5) Console.WriteLine("Not Weird");
        else if(n >= 6 && n <= 20) Console.WriteLine("Weird");
        else if(n > 20) Console.WriteLine("Not Weird");
    }
}

File: Practice/AllDomains/Tutorials/30DaysOfCode/Day3IntrotoConditionalStatements.js
/*	author:@shivkrthakur  */
process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = "";
var input_stdin_array = "";
var input_currentline = 0;

process.stdin.on('data', function (data) {
    input_stdin += data;
});

process.stdin.on('end', function () {
    input_stdin_array = input_stdin.split("\n");
    main();
});

function readLine() {
    return input_stdin_array[input_currentline++];
}

/////////////// ignore above this line ////////////////////

function main() {
    var N = parseInt(readLine());
    if(N % 2 > 0){
        console.log("Weird");
    }
    else if(N >= 2 && N <= 5){
        console.log("Not Weird");
    }
    else if(N >= 6 && N <= 20){
        console.log("Weird");
    }
    else {
        console.log("Not Weird");
    }
}
